strfry/plugins/pubkey_cache.ts

```diff
@@ -0,0 +1,59 @@
+// In process cache to avoid accepting pubkeys that were already removed from
+// the relay. We store CACHE_MAX_SIZE items in redis so restarts don't lose the
+// cache. This assumes that after CACHE_MAX_SIZE items are processed, the
+// oldest ones are no longer relevant and probably not used anymore in the
+// network to avoid to have an infinite cache.
```

Comment:
I suppose in the case where we want to change our cache size/strategy we do have a record of all requests in the append-only log, right?

Reply:
yes, the vanish_requests stream is unchanged. In fact in case we need it, we could re-run all vanish requests from the beginning